docs/kafka/manage-cluster/README.adoc

```diff
@@ -0,0 +1,250 @@
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-cluster"]
+= Managing {product-long-kafka} on a ROSA cluster
+ifdef::context[:parent-context: {context}]
+:context: deploying-cluster
+
+// Purpose statement for the assembly
+[role="_abstract"]
+
+As an organization administrator, you can deploy {product-long-kafka} within your own controlled environment on your own ROSA cluster.
+
+//Additional line break to resolve mod docs generation error.
+
+[id="proc-installing-on-your-rosa-cluster_{context}"]
+== Installing {product-kafka} on your ROSA cluster
+
+You must install {product-long-kafka} on your {rosa-name} ROSA cluster before users in your organization can create Kafka resources. Use the {product-long-rhoas} `rhoas` command-line interface (CLI) to register your (ROSA) cluster through a call to the Kafka Service Fleet Manager service. This service exposes a Rest API to manage Kafka instances. Registering your ROSA cluster allows the service to install certain necessary add-ons to your cluster so users can provision their Kafka instances.
+
+.Prerequisites
+
+* You have a Red Hat account.
+* You are an organization administrator.
+* You've installed the latest version of the `rhoas` CLI (see {base-url}{installation-guide-url-cli}[Installing and configuring the rhoas CLI^]).
+* You have created a ROSA cluster. {product-long-kafka} uses the bring-your-own-key (BYOK) encryption that you have configured in your ROSA cluster. As a result, volumes used by the service are automatically encrypted using your key. For more information about creating and configuring a ROSA cluster, see https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html[Creating a ROSA cluster with STS using customizations^].
+
+.Procedure
+
+. In a terminal, log in to `rhoas`.
++
+--
+[source,shell]
+----
+$ rhoas login
+----
+
+Your web browser opens the {rh-sso-url}[Red Hat Single Sign-On page^].
+--
+
+. Enter your credentials to log in to your Red Hat account.
++
+--
+Welcome pages in the browser notify you that you've been logged in to `rhoas` successfully.
+
+In your terminal, the `rhoas login` command indicates that you're logged in.
+--
+. In your terminal, register your ROSA cluster using the `openshift-cluster (oc) register-cluster` command.
++
+--
+[source,shell]
+----
+$ rhoas kafka oc register-cluster
+----
+
+This command installs the following add-ons to the cluster:
+
+* Managed Kafka kas-fleetshard-operator: Required for provisioning and managing instances of Apache Kafka on a cluster.
+* Managed Kafka operator
+* Observability operator: Required for metrics collection on the Kafka instance. Unlike the previous two, this add-on appears on the *Projects* page instead of the *Add-ons* page of the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console^].
```

Comment:
```suggestion
* Observability Operator: Required for metrics collection on the Kafka instance. Unlike the previous two, this add-on appears on the *Projects* page instead of the *Add-ons* page of the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console^].
```